content/kit/pi/sd-card.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Flashing your SD card
+---
+
+To fully update your Pi's software, or refresh it if you think it's not working correctly, you can flash our SD card image onto the microSD card in your Pi. 
+
+To update the SD card, you'll need to download our image, `pi-image.img.xz`. The flashing procedure is identical to flashing other Raspberry Pi images.
+
+## Etcher
+We recommend using [etcher](https://etcher.io), as it's simple to use, and available on Windows, macOS and Linux. If you're familiar with Raspberry Pi's and have flashed images before with a different tool, that will also work.
+ 
+![Etcher example](https://etcher.io/static/screenshot.gif)
+
+{{% notice note %}}
+If you choose to use a tool other than Etcher, you may need to extract the `pi-image.img.xz` to `pi-image.img`. There are many tools available for this.
```

Comment:
IMO it would be nice to give an example here, such as [7-zip](http://www.7-zip.org/).